webnn/resources/utils.js

```diff
@@ -67,21 +67,21 @@ const loadTests = (operationName) => {
 };
 
 /**
- * Get exptected data and data type from given resources with output name.
+ * Get exptected suboutput resources from given resources with output name.
  * @param {Array} resources - An array of expected resources
  * @param {String} outputName - An output name
- * @returns {Array.<[Number[], String]>} An array of expected data array and data type
+ * @returns {Object} An object of expected suboutput resources
  */
-const getExpectedDataAndType = (resources, outputName) => {
+const getExpectedSubOutputResources = (resources, outputName) => {
   let ret;
   for (let subResources of resources) {
     if (subResources.name === outputName) {
-      ret = [subResources.data, subResources.type];
+      ret = subResources;
```

Comment:
`subresource` (since "sub" is a prefix rather than a separate word), or just `resource`.